SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetConnection.cpp

```diff
@@ -193,3 +196,114 @@ void USpatialNetConnection::OnHeartbeat()
 {
 	SetHeartbeatTimeoutTimer();
 }
+
+void USpatialNetConnection::InitPing(FTimerManager* InTimerManager, Worker_EntityId InPlayerControllerEntity)
```

Comment:
We should comment somewhere that the ping is global and the pong is per client. It was fairly clear after reading the code but not what I expected when just reading the name.

Reply:
Wait I'm wrong. Deleting this.